step_definitions/gmail_steps.rb

```diff
@@ -0,0 +1,13 @@
+sign_in = LoginVar.new
```

Comment:
Variable should declare into each step definition. We should not use commonly. In future we may create different methods in different classes.

Example:

Then("I should be able to login successfully") do
  sign_in = LoginVar.new
  sign_in.login_btn.click
end